js/content.js

```diff
@@ -243,7 +243,13 @@ function removeBlur() {
 
 // timer for fading in and out (works with both)
 // changing it changes the amount of time it takes to fade out (in ms)
-let timerForFading = 50; 
+const TIMER_FADE_LOOP = 50; 
+// creates variable for how much the image is faded in/out to
+// is 100% since image should disappear or appear completely
+const FADE_PERCENT = 100; 
+// changes the "smoothness" of the fade 
+// smaller means fades in smaller intervals
+const FADE_INTERVAL = 0.1; 
```

Comment:
Can the constants be moved to the top of the file with the rest of them? It makes it easier to find them.